client/app/views/state/state.html

```diff
@@ -6,31 +6,33 @@
     <div class="col-xs-6 text-right">
     </div>
   </div>
+
   <div class="row">
-    <div class="col-xs-10 col-xs-offset-1">
+    <div class="col-sm-10 col-sm-offset-1">
```

Comment:
Don't change these above xs since we don't have any other layouts. This isn't a responsive site and not having col set at the lower breakpoint totally breaks the layout. I'd rather have people have to scroll through a squished layout then have one that is non-functional.